src/utils/spp.spec.ts

```diff
@@ -0,0 +1,49 @@
+
+import assert from 'assert';
+import sinon from 'sinon';
+import { spp } from './spp.js';
+import { sinonUtil } from './sinonUtil.js';
+import request from '../request.js';
+
+describe('utils/spp', () => {
+  const siteUrl = 'https://contoso.sharepoint.com';
+  afterEach(() => {
+    sinonUtil.restore([
+      request.get
+    ]);
+  });
+
+  after(() => {
+    sinon.restore();
+  });
+
+  it('calls api correctly and returns false if site is not a content center', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts: any) => {
+      if ((opts.url as string).indexOf(`/_api/web?$select=WebTemplateConfiguration`) > -1) {
```

Comment:
Same here, let's remove `as string` and let's use an exact match with the URL instead of `indexof`. Let's apply this to all tests.

Reply:
Done